src/components/VideoList/video-list.css

```diff
@@ -0,0 +1,38 @@
+.video-list {
+  --width: calc(250px * 1);
+  margin: auto;
+  margin-top: 4rem;
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  max-width: calc(100% - (100% / var(--width)));
+}
+
+@media (max-width: 768px) {
+  .video-list {
+    margin: 4rem 0;
+  }
+}
+
+
+
+
+
+
+/* @media (max-width: 1200px) {
```

Comment:
Please take a look at this file and you can consider to remove the commented code from here.